test-jw/src/adEngineMock.js

```diff
@@ -0,0 +1,39 @@
+import { communicationService } from "./communication/communicationService";
```

Comment:
can't we use communication service from jwplayer-fandom package? 🤔

Reply:
I'm not sure. It's not exported to the main package and this way we imitate more the actual apps that create a similar wrappers around https://github.com/Wikia/post-quecast 🤔 It's also too late for me to make it work 😅